instagram-app/src/components/PostContainer/PostContainer.js

```diff
@@ -7,16 +7,31 @@ import CommentSection from '../CommentSection/CommentSection';
 const PostContainer = props => {
     return (
         <div className="full-post">
-            <Post data={props.data} />
-            <CommentSection comments={props.comments} addComment={props.addComment}/>
+            
+            <Post data={props.data} 
+            likes={props.likes} 
+            handleLikes={props.handleLikes} 
+            index={props.index}/>
+            
+            <CommentSection 
+            comments={props.comments} 
+            data={props.data}
+            addComment={props.addComment} 
+            handleChange={props.handleChange}
+            key= {props.index}
+            likes={props.likes}
+            index={props.index}/>
         </div>
     );
 }
```

Comment:
Line 23 - it might be more helpful to also specify in the prop name what kind of index the `index` prop refers to (perhaps rename it to something like `postIndex`).